build-system/tasks/e2e/index.js

```diff
@@ -126,17 +125,13 @@ async function e2e() {
       });
     }
 
-    mocha.run(failures => {
+    mocha.run(async failures => {
       // end web server
       cleanUp_();
 
       // end task
-      if (failures) {
-        process.exitCode = 1;
-        rejecter();
-      }
-      process.exit();
-      resolver();
+      process.exitCode = failures ? 1 : 0;
+      await resolver();
```

Comment:
qq, does returning the resolver here also fix it or no?

Reply:
Turns out the local test server wasn't shut down properly and was keeping the job alive. So I fixed that but now the Travis build VM hangs :/

Reply:
@rsimha `stdio: 'ignore'` did the trick! Thanks for your help. Will merge this and will consolidate e2e and visual-diff's webserver functions in a future PR.